null_safety/calculate_lix/.idea/workspace.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You can remove this file as well - the workspace file is just user specific settings.